uamqp/async_ops/client_async.py

```diff
@@ -517,7 +520,9 @@ def __init__(
         return True
 
     async def _transfer_message_async(self, message, timeout):
-        sent = await asyncio.shield(self.message_handler.send_async(message, self._on_message_sent, timeout=timeout))
+        sent = await asyncio.shield(
+            self.message_handler.send_async(message, self._on_message_sent, timeout=timeout),
```

Comment:
loop threadthrough?

Reply:
loop is not used in `send_async`.
if you mean `asyncio.shield` it has already been added.